src/cmd_line.h

```diff
@@ -28,4 +28,10 @@ void cmd_line_init();
  */
 void cmd_line_show_prompt();
 
+/**
+ * @brief Run a designated test user process
```

Comment:
np: by saying a designated user process, it sounds like you can give a program location to this function, when in fact it is always the same user program